CONTRIBUTING.md

```diff
@@ -24,14 +24,15 @@ Before you contribute a new feature, consider submitting an Issue to discuss the
 
 ## How to Contribute Changes
 
-First, fork this repository to your own GitHub account. Create a new branch that describes your changes (i.e. `line-counter-docs`). Push your changes to the branch on your fork and then submit a pull request to this repository.
+First, fork this repository to your own GitHub account. Create a new branch that describes your changes (i.e. `line-counter-docs`). Push your changes to the branch on your fork and then submit a pull request to `develop` branch of this repository.
```

Comment:
We could also add some instructions describing how to install `supervision` locally. We could link to our installation guide. https://roboflow.github.io/supervision/

Reply:
We could also mention that `supervision` has a backlog of features that we plan to add and from which you can choose. https://github.com/orgs/roboflow/projects/5/views/1